kiali-server/templates/deployment.yaml

```diff
@@ -123,6 +123,9 @@ spec:
           value: "{{ .Values.deployment.logger.time_field_format }}"
         - name: LOG_SAMPLER_RATE
           value: "{{ .Values.deployment.logger.sampler_rate }}"
+        {{- with .Values.deployment.extraEnvs }}
```

Comment:
For legacy reasons, all settings are snake_case, not camelCase. To remain consistent across all Kiali settings in the Kiali CR and server helm charts, use snake_case for any new settings.

Also to remain consistent with other analogous settings (such as existing settings of [`additional_labels`](https://github.com/kiali/helm-charts/blob/v1.84.0/kiali-server/values.yaml#L61) and [`additional_service_yaml`](https://github.com/kiali/helm-charts/blob/v1.84.0/kiali-server/values.yaml#L44)), extra_envs should be renamed `additional_envs`.

Reply:
Also, [as per the developer checklist mentioned earlier](https://github.com/kiali/kiali-operator/blob/master/DEVELOPING.adoc#are-you-altering-a-kiali-server-configuration-setting), "Ensure each group of settings are alphabetically sorted in all the files where the new setting was added."

So this new `additional_envs` should be placed on the line before [`additional_service_yaml` found here](https://github.com/kiali/helm-charts/blob/v1.84.0/kiali-server/values.yaml#L44).